internal/cluster/validations/validations.go

```diff
@@ -905,3 +911,23 @@ func ValidateHighAvailabilityModeWithPlatform(highAvailabilityMode *string, plat
 
 	return nil
 }
+
+func ValidateIgnitionImageSize(config string) error {
+	var err error
+	var data *bytes.Reader
+
+	// Ensure that the ignition archive isn't larger than 256KiB
+	configBytes := []byte(config)
+	content := isoeditor.IgnitionContent{Config: configBytes}
+	data, err = content.Archive()
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("Failed to create ignition archive (config: %s)", config))
```

Comment:
Not sure we want to dump the entire ignition config in this error. Maybe in a log, but not in the error.

Reply:
Yeah probably too much. I'll just remove it.